cpp/oneapi/dal/graph/undirected_adjacency_array_graph.cpp

```diff
@@ -134,31 +138,34 @@ template class ONEDAL_EXPORT undirected_adjacency_array_graph<empty_value,
                                                               std::int32_t,
                                                               std::allocator<char>>;
 
+template struct ONEDAL_EXPORT graph_traits<undirected_adjacency_array_graph<empty_value,
```

Comment:
Explicit instantiation is useless for `graph_traits`, there is no any method defined in `.cpp`